src/python/genny/genny_auto_tasks.py

```diff
@@ -230,67 +253,82 @@ def construct_task_json(workloads, variants):
 
 def main():
     """
-    Main Function: parses args, writes to file evergreen tasks in json format for workloads that are specified in args.
+    Main Function: parses args, writes a JSON file representing *either* evergreen tasks or buildvariants that can be passed to generate.tasks.
+    If run with --generate-all-tasks, this program outputs a JSON file of task definitions for all workloads in the /src/workloads genny directory.
```

Comment:
Maybe also add a phrase to say this is to get around limitations of generate.tasks

Reply:
Done.